### PR TITLE
Review My Project from Trials

### PR DESCRIPTION
**Pull request by Piercel2022**
In this pull request , I am playing the role of the reviewer to detect some irregularities in the code I write from Microverse Trials.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+</head>
+<body>
+    <nav id="navbar">
+        <header>JS Documentation</header>
+<ul>
+   
+    <li><a class="nav-link" href="#Introduction" >Introduction</a></li>
+    <li><a class="nav-link" href="What_you_should_already_know" >What you should already know</a></li>
+    <li><a class="nav-link" href="JavaScript_and_Java" >JavaScript and Java</a></li>
+    <li><a class="nav-link" href="Hello_world" >Hello world</a></li>
```
Comment (Piercel2022) [inline index.html:18]: Kindly add the right symbol to the href to follow the link

File: index.html [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    <li><a class="nav-link" href="#Variables" >Variables</a></li>
+    <li><a class="nav-link" href="declaring_variables.html" >Declaring variables</a></li>
+    <li><a class="nav-link" href="Variable_scope" >Variable scope</a></li>
+    <li><a class="nav-link" href="Global_variables" >Global variables</a></li>
```
Comment (Piercel2022) [inline index.html:22]: Kindly add the right symbol to the href attribute to get the right link

File: index.html [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+    <li><a class="nav-link" href="#Constants" >Constants</a></li>
+    <li><a class="nav-link" href="Data_types" >Data types</a></li>
+    <li><a class="nav-link" href="if...else statement" >if...else statement</a></li>
+    <li><a class="nav-link" href="Function_declarations" >while statement</a></li>
+    <li><a class="nav-link" href="Function_declarations" >Function declarations</a></li>
```
Comment (Piercel2022) [inline index.html:27]: kindly add the right symbol to the href attribute to follow the link

**Review by Piercel2022 — COMMENTED** [review]
# Require changes :recycle: 

Please check my comments below and make all required changes.

Submit for a review when you are done.

Thanks :grin:

File: index.html (hunk 1 above)
Comment (zhadier) [reply index.html:18]: - [ ] Hey @Piercel2022, Great job finding this mistake but you seem to have missed pointing out why and how they should fix it. It'll make your Review process much more meaningful if you add the reason behind your proposed changes and give an example (a link) to help them understand what they have to do. Please do that for all your comments 🌥️ 
- [ ] Additionaly, you can group all of these links together in one Review comment using GitHub [inline comment features](https://github.blog/changelog/2020-02-21-a-new-interaction-for-multi-line-pull-request-comments/). Kindly do that as posting the same Review multiple times can make the Review longer than necessary :)
- [ ] Kindly ensure you find at least two different and unique things to Review, so we can have a better look at what you can do to improve your Code Reviews :)

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @Piercel2022  👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
✔️ Good Communication Flow
✔️ Great use of GH Inline features

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <title>Tehnical Documentation</title>
+</head>
+<body>
+    <nav id="navbar">
+        <header>JS Documentation</header>
```
Comment (Piercel2022) [inline index.html:12]: For readability, add only two spaces of indentation. Don't use the Tab key for that

File: index.html [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+<section class="main-section" id="Data_types"><header>Data types</header><article>
+    <p>The latest ECMAScript standard defines seven data types:</p>
+    <ul>
+        <li><p>Six data types that are primitives:</p>
+            <ul>
+                <li>Boolean. true and false.</li>
+                <li>null. A special keyword denoting a null value. Because JavaScript is case-sensitive, null is not the same as Null, NULL, or any other variant. </li>
+                <li>undefined. A top-level property whose value is undefined</li>
+                <li>Number. 42 or 3.14159.</li>
+                <li>String. "Howdy"</li>
```
Comment (Piercel2022) [inline index.html:110]: For readability add only two spaces for indentation. Kindly remove the useless spaces

File: index.html [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+<section class="main-section" id="Reference"><header>Reference</header><article>
+    <ul><li>All the documentation in this page is taken from <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide" target="blank">MDN</a></li></ul>
+</article></section>
+
+ 
+
+
+
+
+
```
Comment (Piercel2022) [inline index.html:174]: For readability of the code, kindly remove these lines in your code. These lines are unnecessary for readability.

File: index.html (hunk 3 above)
Comment (Piercel2022) [reply index.html:27]: You forgot to add (#) to the href attribute, kindly add that symbol to get the required link

File: index.html (hunk 2 above)
Comment (Piercel2022) [reply index.html:22]: You forgot to add the symbol (#) to the href attribute, kindly add that symbol to have the link as you expect

**Review by Piercel2022 — COMMENTED** [review]
# Requires changes :recycle: 

Check my comments to make the required changes!

Thanks :grin:

**Review by zhadier — APPROVED** [review]
## Hi @Piercel2022 👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 



### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**